include/modem/lte_lc.h

```diff
@@ -1343,25 +1343,108 @@ int lte_lc_normal(void);
  * Set modem PSM parameters.
  *
  * Requested periodic TAU (RPTAU) and requested active time (RAT) are used when PSM mode is
- * subsequently enabled using lte_lc_psm_req().
+ * subsequently enabled using lte_lc_psm_req(). These are the requested values and network
+ * determines the final values.
  *
- * For reference see 3GPP 27.007 Ch. 7.38.
+ * For encoding of the variables, see nRF AT Commands Reference Guide, 3GPP 27.007 Ch. 7.38., and
+ * 3GPP 24.008 Ch. 10.5.7.4a and Ch. 10.5.7.3.
  *
- * @param[in] rptau Requested periodic TAU as a null-terminated string.
- *                  Set to @c NULL to use manufacturer-specific default value.
- * @param[in] rat Requested active time as a null-terminated string.
- *                Set to @c NULL to use manufacturer-specific default value.
+ * @param[in] rptau
+ * @parblock
+ *          Requested periodic TAU as a null-terminated 8 character long bit field string.
+ *          Set to @c NULL to use modem's default value.
+ *          See 3GPP 24.008 Ch. 10.5.7.4a for data format.
+ *
+ *          For example, value of 32400 s is represented as '00101001'.
+ *
+ *          Bits 5 to 1 represent the binary coded timer value that is multiplied by timer unit.
+ *
+ *          Bits 8 to 6 define the timer unit as follows:
+ *
+ *          - 000: 10 minutes
+ *          - 001: 1 hour
+ *          - 010: 10 hours
+ *          - 011: 2 seconds
+ *          - 100: 30 seconds
+ *          - 101: 1 minute
+ *          - 110: 320 hours
+ *          - 111: Timer is deactivated
```

Comment:
As far as I understand, `111` should be removed from the description because TAUs can not be disabled. I think that `111` for T3412_EXT just means that T3412 is used instead. `111` makes sense in the %XMONITOR notification, but not here, I think.

Reply:
Line with '111' removed.